src/odemis/driver/andorcam2.py

```diff
@@ -3226,6 +3335,31 @@ def AbortAcquisition(self):
         self.status = AndorV2DLL.DRV_IDLE
         self.acq_aborted.set()
 
+    def GetNumberNewImages(self, p_first, p_last):
```

Comment:
Docstring for arguments at least?

Reply:
This simulates the Andor2 SDK C API. The python types are quite messy (like `POINTER(c_uint32)`), and don't really match the API documentation, so I'd rather not use type annotation. I could add the official C types, like `long*` .